### ISSUE TITLE
Builds for Debian

### ISSUE THREAD

**Issue by rugk** [RicinApp/Ricin#153]
Currently online Ubuntu builds are provided, but "GNU/Linux" != "Ubuntu".

You may also just use the tox.chat repositories for this.
Or maybe offer a statically linked version if it is not too large.

**Comment by SkyzohKey** [comment]
There is an AppImage build, which should works on every Linux distro too, and I plan to make .rpm's  
Static builds could be a solution yep

**Comment by rugk** [comment]
I'd rather argue for Flatpak & Snap, which might be the distribution features (with sandboxing) of the future.

**Comment by SkyzohKey** [comment]
@rugk Flatpak needs a runtime, Snap too. AppImage is a standard format that runs everywhere without any dependencies needed, that's why I prefer it ;)

**Comment by rugk** [comment]
Yeah, just publishing multiple formats would be great IMHO, so that everyone can use it. And if we use such container technologies we not only have a security enhancement, but there are also only 3 technologies to use (instead of the many .deb/.rpm/… package formats distros may have).

**Comment by Encrypt** [comment]
Just to be sure @rugk , when you say there is no package for "Debian", you mean Wheezy / Jessie, right?
Because there are actually packages for Stretch and Sid: https://build.tox.chat/view/ricin/

I couldn't create Ricin packages for Wheezy and Jessie because @SkyzohKey uses "cutting edge technologies" whose packages are not available for Wheezy / Jessie or too old at least.

**Comment by rugk** [comment]
Ah okay, thanks for your information. In fact Wheezy is really outdated, but it would be nice if you could support Jessie. Maybe use jessie-backports?
